### PR TITLE
Fix display issues with add to cal widget

### PR DESCRIPTION
**Pull request by erikdstock**
An auctions team member reported that the [CMA Benefit Auction 2017](https://www.artsy.net/auction/cma-benefit-auction-2017) partner was complaining that its link was unclickable. I looked into it and saw that the calendar wrapper was invisible but blocking the link from being clickable. In addition the hover events were not revealing the calendar bits that belonged there. This is an initial, quick fix that seems to resolve it, but I'm not 💯 on whether this is the best way to go about it or whether it will affect other areas the widget might be used.

Before: Text/link not directly under mouse when they are below the widget, widget hover function not working
caldiv

Now: Both are working
caldiv2

### CODE REVIEW

File: desktop/components/add_to_calendar/react.js [hunk 1]
```diff
@@ -17,8 +17,10 @@ export default class AddToCalendar extends Component {
 
     this.setState(() => {
       this.view = new AddToCalendarView({
-        el: this.props.el || this.$('.auction2-AuctionInfo__callout')
+        el: this.props.el || this.$('.add-to-calendar')
       })
+      console.log("*****", this.view)
```
Comment (sweir27) [inline desktop/components/add_to_calendar/react.js]: 🚧

**Comment by sweir27** [comment]
The other place this component is used is on the "fair events page":
image

You can check one out here: https://staging.artsy.net/sunday-2017/info/events. I would make sure this still behaves as expected.

**Comment by erikdstock** [comment]
Ah thanks! Looks good. 
image

**Comment by craigspaeth** [comment]
There's probably an easier way to do this with just CSS, e.g. make the menu a child of the button and use `:hover` to set it to set `display: block`—happy to chat more irl about that. I won't let that block this PR though as given the existing JS view, this seems like a fine fix 👍  Thanks!

**Comment by erikdstock** [comment]
@craigspaeth I wondered about that but was trying to achieve what I thought the intent was with minimal changes to the existing structure. Happy to chat today if you want to revisit it.

**Pull request merged**